### ISSUE TITLE
lv_bar indicator area

### ISSUE THREAD

**Issue by mheranco** [lvgl/lvgl#457]
Hi,

great project first of all! 
After updating to version 5.2 (download from littlevgl.com/download) I had an issue with the progress bars. The indicator area does not fill up the whole bar when setting it to its maximum value.

After changing line 320 and 321 in lv_bar_design(...) in lv_bar.c from:

```
indic_area.x2 = (int32_t)((int32_t)w * (ext->cur_value - ext->min_value - 1)) / (ext->max_value - ext->min_value);
indic_area.x2 = indic_area.x1 + indic_area.x2;
```

back to 

```
indic_area.x2 = (int32_t)((int32_t)w * (ext->cur_value - ext->min_value)) / (ext->max_value - ext->min_value);
indic_area.x2 = indic_area.x1 + indic_area.x2 - 1;
```

it is working fine again.
I think this is not intended and the "-1" should not be relative to w (maybe also in the following else statement?).

**Comment by BrianPugh** [comment]
@kisvegabor just fixed this in a482132652152a920ff4767b01824ff071d24895

**Comment by kisvegabor** [comment]
> @kisvegabor just fixed this in a482132

Yes, yesterday I pushed the suggested fix to the `master` branch. It is working now?

**Comment by kisvegabor** [comment]
Great! Thank you for the feedback!
